easysocial/easysocial/group.php

```diff
@@ -83,6 +83,13 @@ public function post()
 		$apiResponse->result	=	new stdClass;
 		$postValues				=	$input->post->getArray();
 
+		// Check EasySocial extension version
+		$component = JComponentHelper::getComponent('com_easysocial');
+		$extension = JTable::getInstance('extension');
+		$extension->load($component->id);
+		$manifest = new Joomla\Registry\Registry($extension->manifest_cache);
```

Comment:
@Pratikshakhandagale  - 
first import this
jimport( 'joomla.registry.registry' );

then  create object by  - 
new JRegistry($data);